## 데이터 불러오기

In [38]:
import pandas as pd
pd.set_option('display.max_columns',50)
pd.set_option('display.float_format', '{:.4f}'.format)

df = pd.read_csv("https://raw.githubusercontent.com/Datamanim/datarepo/main/nba/nba.csv",encoding='latin',sep=';')
df.head()

Rk             Player Pos  Age   Tm   G  GS      MP     FG     FGA    FG%  \
0   1   Precious Achiuwa   C   22  TOR  73  28 23.6000 3.6000  8.3000 0.4390   
1   2       Steven Adams   C   28  MEM  76  75 26.3000 2.8000  5.1000 0.5470   
2   3        Bam Adebayo   C   24  MIA  56  56 32.6000 7.3000 13.0000 0.5570   
3   4       Santi Aldama  PF   21  MEM  32   0 11.3000 1.7000  4.1000 0.4020   
4   5  LaMarcus Aldridge   C   36  BRK  47  12 22.3000 5.4000  9.7000 0.5500   

      3P    3PA    3P%     2P     2PA    2P%   eFG%     FT    FTA    FT%  \
0 0.8000 2.1000 0.3590 2.9000  6.1000 0.4680 0.4860 1.1000 1.8000 0.5950   
1 0.0000 0.0000 0.0000 2.8000  5.0000 0.5480 0.5470 1.4000 2.6000 0.5430   
2 0.0000 0.1000 0.0000 7.3000 12.9000 0.5620 0.5570 4.6000 6.1000 0.7530   
3 0.2000 1.5000 0.1250 1.5000  2.6000 0.5600 0.4240 0.6000 1.0000 0.6250   
4 0.3000 1.0000 0.3040 5.1000  8.8000 0.5780 0.5660 1.9000 2.2000 0.8730   

     ORB    DRB     TRB    AST    STL    BLK    TOV     PF     PTS  
0 2.0000 4.5000  6.5000 1.1000 0.5000 0.6000 1.2000 2.1000  9.1000  
1 4.6000 5.4000 10.0000 3.4000 0.9000 0.8000 1.5000 2.0000  6.9000  
2 2.4000 7.6000 10.1000 3.4000 1.4000 0.8000 2.6000 3.1000 19.1000  
3 1.0000 1.7000  2.7000 0.7000 0.2000 0.3000 0.5000 1.1000  4.1000  
4 1.6000 3.9000  5.5000 0.9000 0.3000 1.0000 0.9000 1.7000 12.9000

## 1. Tm 컬럼은 각 팀의 이름을 의미한다. TOR팀의 평균나이를 소수 4째 자리까지 구하여라

## 2. Pos 컬럼은 포지션을 의미한다. 전체 선수 중 최소나이대의 선수들을 필터하고 그들 중 가장 많은 포지션은 무엇인지 확인하라

## 3. 선수들의 이름은 first_name+ 공백 + last_name으로 이루어져 있다. 가장 많은 first_name은 무엇이며 몇 회 발생하는지 확인하라

## 4. PTS컬럼은 경기당 평균득점수 이다. 각포지션별로 경기당 평균득점수의 평균을 구하여라

## 5. G컬럼은 참여한 경기의 숫자이다. 각 팀별로 가장 높은 경기참여 수를 가진 선수들의 경기 참여 숫자의 평균을 구하여라

## 6. Tm의 값이 MIA이며 Pos는 C또는 PF인 선수의 MP값의 평균은

## 7. 전체 데이터중 G의 평균값의 1.5배 이상인 데이터들만 추출했을때 3P값의 평균은?

## 8.  Age의 평균 이상인 그룹과 평균 미만인 그룹간의 G값의 평균의 차이는?

## 9. 평균나이가 가장 젊은 팀은 어디인가

## 10. Pos그룹별 평균 MP값을 구하여라